contracts/hackatom/src/contract.rs

```diff
@@ -13,6 +13,11 @@ pub struct InitMsg {
     pub beneficiary: HumanAddr,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct MigrateMsg {
+    pub verifier: HumanAddr,
```

Comment:
I'd appreciate a doc comment here, explaining how this field is used

Reply:
Will add. If this addresses all concerns, please merge it.